### DISCUSSION TITLE
Project database - Spectrum - Reference files

### DISCUSSION THREAD

**Discussion by majlomilo** [Ifsttar/I-Simpa#349] [General]
I couldn't find any information about these names in the manual. White Noise and Pink Noise is a matter of course. But what do they mean: BBDR_TR, BBDR_VL, BBSG_TR, BBSG_VL, ES_TR, ES_VL?????

please help

**Comment by nicolas-f** [comment]
Its the emission spectrum of light vehicle (VL is véhicule léger in french) or heavy vehicle (TR is train routier french,) on various road pavements such as BBDR for porous asphalt concrete or BBSG for Bituminous concrete
Semi-Grained, ES for surface coating.

**Reply by majlomilo** [reply]
Thank you for your help.
